rke/k8s_rke_system_images.go

```diff
@@ -6988,7 +6988,7 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			CanalNode:                 "rancher/mirrored-calico-node:v3.21.1",
 			CanalCNI:                  "rancher/mirrored-calico-cni:v3.21.1",
 			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.21.1",
-			CanalFlannel:              "rancher/mirrored-coreos-flannel:v0.15.1",
+			CanalFlannel:              "rancher/mirrored-coreos-flannel:v0.17.0",
```

Comment:
we would also need to bump Flannel and FlannelCNI 
```
Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
FlannelCNI:             "rancher/flannel-cni:v0.3.0-rancher6",
```			
these would use the new flannel template so after updating it, we might need one round of testing to confirm flannel works as expected.

Reply:
based on an older conversation with @luthermonson, it should be okay to replace `flannel-cni:v0.3.0-rancher6` with new tag upstream. that's what upstream uses and we can be consistent with it unless we see any blocker issues with it.